assets/dev/js/editor/utils/helpers.js

```diff
@@ -704,4 +704,10 @@ module.exports = {
 	sanitize( value, options ) {
 		return DOMPurify.sanitize( value, options );
 	},
+
+	sanitizeUrl( url ) {
+		const isValidUrl = !! url ? isValidAttribute( 'a', 'href', url ) : false;
```

Comment:
@louiswol94 @mykytamurzin 

I have added a general validation of the `url` parameter. Dompurify was throwing an error when we had an empty value.